src/AbstractClient.php

```diff
@@ -5,20 +5,45 @@
 namespace CommentSold;
 
 use CommentSold\Clients\Rest;
+use CommentSold\Enum\Environment;
+use CommentSold\Exception\InvalidArgumentException;
 
 abstract class AbstractClient
 {
     protected Rest $restClient;
+    protected Environment $environment = Environment::PRODUCTION;
```

Comment:
Defaulting to sandbox means that you always need to set the environment when using this in production. I thought that might be more of a pain, although I guess if you are writing your code to work with both you'd have the line there anyway and just switch based on your local environment. 🤔 

Do you think it will be obvious (I'll add it to the README.md) to a user?

Reply:
Well, it's up to you. I was just thinking that virtually everyone will use sandbox first, so the code will always have to be written. And, chances are, they will always need to have the line in there based on their own .env. I just thought defaulting to a safe env might be preferred.